tests/Tests/Payload/SearchMessagesPayloadResponseTest.php

```diff
@@ -39,8 +39,8 @@ protected function assertResponse(array $responseData, PayloadResponseInterface
     {
         $fileResult = $payloadResponse->getResult();
 
-        $this->assertInstanceOf('CL\Slack\Model\MessageResult', $fileResult);
-        $this->assertCount(1, $fileResult->getMatches());
+        self::assertInstanceOf('CL\Slack\Model\MessageResult', $fileResult);
+        self::assertCount(1, $fileResult->getMatches());
 
         foreach ($fileResult->getMatches() as $x => $match) {
             $this->assertMessageResultItem($responseData['messages']['matches'][$x], $match);
```

Comment:
How come that this `$this->assert...` is not a `self::assert...` too?

Reply:
Because the method is not static. I can adjust that too